pom.xml

```diff
@@ -480,6 +482,16 @@
                 <artifactId>typetools</artifactId>
                 <version>${typetools.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.code.gson</groupId>
+                <artifactId>gson</artifactId>
+                <version>${gson.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.yaml</groupId>
+                <artifactId>snakeyaml</artifactId>
+                <version>${snakeyaml.version}</version>
```

Comment:
One of the downsides of building a testing framework is that you can't use all the popular libraries. There are ways to work around this. We'll come to that later.

Reply:
Understood. However, if I base the PR on `cucumber-config` instead as suggested these libraries will come as transitive dependencies. What's the reason for not having them (footprint, Java version limitations)? What's the alternative you suggest?

Reply:
Cucumber is a test library. Adding dependencies means that those dependencies are also present on the class path when tests are executed. Now depending on the build system used, the order in the dependency declaration it may also occur that our transitive dependencies replace the transitive dependencies of the project we're testing. This is highly undesirable.

We can resolve it by adding the dependency with scope  `provided` and checking if the class we're interested in is available on the class path before creating an instance of it.

Reply:
We’ll shade those deps, created a ticket in the monorepo